src/pages/Home.tsx

```diff
@@ -35,6 +35,8 @@ const Home: FC = () => {
   const [txError, setTxError] = useState("");
 
   const theme = darkMode ? "dark" : "light";
+  const lnImplAvailable =
```

Comment:
Could name it `isLnImplAvailable` to indicate boolean more transparent

Reply:
Good idea, thank you! :)

Done :+1: